pkg/presets/presets.go

```diff
@@ -1,5 +1,7 @@
 package presets
 
+var IPNIAnnounceURLs = []string{"https://cid.contact/announce"}
```

Comment:
I think we could also have a preset for the IPNI URL and make the `IPNI_ENDPOINT` env var not required.